data/bundles.yml

```diff
@@ -63,3 +63,8 @@ questions:
             - {value: "$container->get('kernel')->getBundles()",          correct: true}
             - {value: "$container->getParameter('bundles')",              correct: false}
             - {value: "$container->getParameter('application.bundles')",  correct: false}
+    -
+        question: 'True or False ? the entry point of a Bundle MUST extends `Symfony\HttpKernel\Bundle\Bundle` class.'
```

Comment:
Agree,  but the docs never mention it.  I  guess no one implement the interface in Symfony framework (we do it in BackBee CMS)